src/components/application_manager/include/application_manager/application_manager_impl.h

```diff
@@ -837,8 +837,10 @@ class ApplicationManagerImpl
   /**
    * @brief ForbidStreaming forbid the stream over the certain application.
    * @param app_id the application's id which should stop streaming.
+   * @param service_type Service type to check
```

Comment:
@SNiukalov maybe remove description here?

Reply:
@ydementieiev 
Thanks, fixed in:https://github.com/smartdevicelink/sdl_core/pull/3293/commits/51915151492980976de1ad212161d71d52e6bbfc